packages/web3-utils/src/errors.ts

```diff
@@ -155,3 +154,33 @@ export class InvalidTypeAbiInput extends Web3Error {
 		super(value, 'components found but type is not tuple');
 	}
 }
+
+// TODO Type of value for all errors should be any as
+// any value can be passed when an error occurs
```

Comment:
I believe the type of `value` for all errors should be `any` as we are throwing an error because we got an unexpected value, therefore it could be anything. Thoughts?

Reply:
Yes `any` is fine for sepcific cases, but if we can use `unknown` instead its much safer as the end user can use `as` keyword with the value.

Reply:
https://github.com/ChainSafe/web3.js/issues/4583